### PR TITLE
fixes for restore corner cases

### PR DESCRIPTION
**Pull request by Slach**
- add integration test for MATERIALIZED VIEW + .inner table
- change sort order during `restore --rm`, I hope it allow to resolve depency object restore

### CODE REVIEW

File: pkg/clickhouse/clickhouse.go [hunk 1]
```diff
@@ -488,11 +503,16 @@ func (ch *ClickHouse) CreateDatabase(database string) error {
...
+	var isAtomic bool
+	var err error
+	if isAtomic, err = ch.IsAtomic(table.Database); err != nil {
 		return err
 	}
 	if dropTable {
-		query := fmt.Sprintf("DROP TABLE IF EXISTS `%s`.`%s` NO DELAY", table.Database, table.Name)
+		query := fmt.Sprintf("DROP TABLE IF EXISTS `%s`.`%s`", table.Database, table.Name)
+		if isAtomic {
+			query += " SYNC"
```
Comment (Slach) [inline pkg/clickhouse/clickhouse.go]: NO DELAY here
